packages/store/src/commands/store/copy.ts

```diff
@@ -79,6 +81,19 @@ export default class Copy extends BaseBDCommand {
     }
   }
 
+  private async validateNodeVersion(): Promise<void> {
+    const nodeVersion = process.versions.node
+    const nodeMajorVersion = Number(nodeVersion.split('.')[0])
+
+    if (nodeMajorVersion < 20) {
+      renderError({
+        headline: 'Update to Node 20 or higher to run `shopify store copy`.',
+        body: [`The \`store copy\` command requires Node 20 or higher. Current Node version: ${nodeMajorVersion}.`],
+      })
+      process.exit(1)
```

Comment:
try to avoid using `process.exit` and instead throw an error, that will be tracked.

Also, in `cli-kit` we have the `exitIfOldNodeVersion` already. It's currently checking for node <18, but maybe we can upgrade it to <20